src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1101,11 +1101,18 @@
         <summary>SyntaxToken representing the optional ref or out keyword.</summary>
       </PropertyComment>
     </Field>
-    <Field Name="Expression" Type="ExpressionSyntax">
+    <Field Name="Expression" Type="ExpressionSyntax" Optional="true">
       <PropertyComment>
         <summary>ExpressionSyntax node representing the argument.</summary>
       </PropertyComment>
     </Field>
+    <Field Name="Type" Type="TypeSyntax" Optional="true"/>
+    <Field Name="Identifier" Type="SyntaxToken" Optional="true">
```

Comment:
This is unpleasant enough a consequence of this approach that we should probably consider a new ExpressionSyntax, even if it is only valid in an out argument. Lets follow up later.